src/main/fc/rc.c

```diff
@@ -772,6 +811,12 @@ bool isMotorsReversed(void)
     return reverseMotors;
 }
 
+#ifdef USE_FEEDFORWARD
+void getFeedforwardAveraging (const pidRuntime_t *pid) {
+    feedforwardAveraging = pid->feedforwardAveraging;
+}
+#endif // USE_FEEDFORWARD
+
```

Comment:
This doesn't make sense (it is not a getter function)
```suggestion
```

Reply:
Sorry I didn't know how otherwise to get it - your suggestion below is perfect!